### PR TITLE
fix silent error when instantiating chaincode

### PR DESCRIPTION
**Pull request by sgbenav**
Fix the silent error: [ Error parsing buffer to JSON ], when Instantiating chaincode.

### CODE REVIEW

File: @worldsibu/convector-common-fabric-helper/src/client.helper.ts [hunk 1]
```diff
@@ -186,10 +186,8 @@ export class ClientHelper {
     let result: any = proposalResponse.proposalResponses[0].response.payload;
 
     try {
-      result = Transform.bufferToObject(result);
-    } catch (err) {
-      result = Transform.bufferToString(result);
-    }
+      result = JSON.parse(result);
+    } catch (err) { }
```
Comment (diestrin) [inline @worldsibu/convector-common-fabric-helper/src/client.helper.ts]: Linter fails here. No empty code block allowed. Put a comment that this error is expected and harmless.

**Comment by diestrin** [comment]
Issue #14

File: @worldsibu/convector-common-fabric-helper/src/client.helper.ts [hunk 2]
```diff
@@ -187,7 +186,9 @@ export class ClientHelper {
 
     try {
       result = JSON.parse(result);
-    } catch (err) { }
+    } catch (err) {
+      // This error is expected and harmless
```
Comment (sgbenav) [inline @worldsibu/convector-common-fabric-helper/src/client.helper.ts:190]: @diestrin fixed

File: @worldsibu/convector-common-fabric-helper/src/client.helper.ts (hunk 1 above)
Comment (sgbenav) [reply @worldsibu/convector-common-fabric-helper/src/client.helper.ts]: fixed

**Comment by diestrin** [comment]
Thanks. I'll test and merge if works. Planned to be release next week

**Pull request merged**